### PR TITLE
Fixed bug csv 'Total' not showing up.

### PR DESCRIPTION
**Pull request by eoliveros**
Original format of displaying the Total is still the same from the app point of view.

When exported to csv for reporting, I changed it to:
- two new format columns for exporting csv. 
   - Total
   - Claimed

So the changes only applies for the exporting of CSV.

### CODE REVIEW

File: zapd/models.py [hunk 1]
```diff
@@ -315,11 +315,45 @@ def _format_total_column(view, context, model, name):
...
+        if model.status == model.STATE_DECLINED:
+            return Markup('-')
+        total = 0
+        total_claimed = 0
+        for payment in model.payments:
+            total += payment.amount
+            if payment.status == payment.STATE_SENT_FUNDS:
+                total_claimed += payment.amount
+        total = str(total / 100)
+        #total_claimed = total_claimed / 100
```
Comment (djpnewton) [inline zapd/models.py]: remove dead code

**Comment by djpnewton** [comment]
do you think we should make the columns the same for viewing and exporting? it might actually make it a bit clearer?

**Comment by eoliveros** [comment]
Hmm... I actually like the way that its displaying it from the app(one column) but multiple value. If we wanted to know it, they we click on the link to drill into the info. But maybe the finance and Amy will want to separate it so I'm not sure. Probably show them first and see what they say.

**Comment by djpnewton** [comment]
ok good point, lets leave it for now

just fix the dead code and I will merge

**Comment by eoliveros** [comment]
> ok good point, lets leave it for now
> 
> just fix the dead code and I will merge

changed it from "#" to "". also turned the result to string for comparison.

File: zapd/models.py [hunk 2]
```diff
@@ -315,11 +315,45 @@ def _format_total_column(view, context, model, name):
...
+        total_claimed = 0
+        for payment in model.payments:
+            total += payment.amount
+            if payment.status == payment.STATE_SENT_FUNDS:
+                total_claimed += payment.amount
+        total = str(total / 100)
+        total_claimed = str(total_claimed / 100)
+        payments_url = url_for('.payments_view', proposal_id=model.id)
+        if total_claimed != total:
+            total_claimed=total_claimed
```
Comment (djpnewton) [inline zapd/models.py]: this does not actually do anything ??

File: zapd/models.py [hunk 3]
```diff
@@ -315,11 +315,45 @@ def _format_total_column(view, context, model, name):
...
+        total_claimed = 0
+        for payment in model.payments:
+            total += payment.amount
+            if payment.status == payment.STATE_SENT_FUNDS:
+                total_claimed += payment.amount
+        total = str(total / 100)
+        total_claimed = str(total_claimed / 100)
+        payments_url = url_for('.payments_view', proposal_id=model.id)
+        if total_claimed == total:
+            total=total
```
Comment (djpnewton) [inline zapd/models.py]: this does nothing

File: zapd/models.py [hunk 4]
```diff
@@ -315,11 +315,45 @@ def _format_total_column(view, context, model, name):
...
+    def _format_total_column_text_total(view, context, model, name):
+        if model.status == model.STATE_DECLINED:
+            return Markup('-')
+        total = 0
+        total_claimed = 0
+        for payment in model.payments:
+            total += payment.amount
+            if payment.status == payment.STATE_SENT_FUNDS:
+                total_claimed += payment.amount
+        total = str(total / 100)
```
Comment (djpnewton) [inline zapd/models.py]: no need to convert to string, better to just compare the integers

File: zapd/models.py [hunk 5]
```diff
@@ -315,11 +315,45 @@ def _format_total_column(view, context, model, name):
...
+    def _format_total_column_text_totalclaimed(view, context, model, name):
+        if model.status == model.STATE_DECLINED:
+            return Markup('-')
+        total = 0
+        total_claimed = 0
+        for payment in model.payments:
+            total += payment.amount
+            if payment.status == payment.STATE_SENT_FUNDS:
+                total_claimed += payment.amount
+        total = str(total / 100)
```
Comment (djpnewton) [inline zapd/models.py]: no need to convert to string, better to just compare the integers

File: zapd/models.py (hunk 2 above)
Comment (eoliveros) [reply zapd/models.py]: suppose to compare total_claimed against total... but i could change it so its just:
    def _format_total_column_text_totalclaimed(view, context, model, name):
        if model.status == model.STATE_DECLINED:
            return Markup('-')
        total = 0
        total_claimed = 0
        for payment in model.payments:
            total += payment.amount
            if payment.status == payment.STATE_SENT_FUNDS:
                total_claimed += payment.amount
        total_claimed = total_claimed / 100
        payments_url = url_for('.payments_view', proposal_id=model.id)
        return Markup(total_claimed)

File: zapd/models.py [hunk 6]
```diff
@@ -315,11 +315,35 @@ def _format_total_column(view, context, model, name):
             '''.format(payments_url=payments_url, total=total, total_claimed=total_claimed)
         return Markup(html)
 
+    def _format_total_column_text_total(view, context, model, name):
+        if model.status == model.STATE_DECLINED:
+            return Markup('-')
+        total = 0
+        total_claimed = 0
```
Comment (djpnewton) [inline zapd/models.py]: unused variable

File: zapd/models.py [hunk 7]
```diff
@@ -315,11 +315,35 @@ def _format_total_column(view, context, model, name):
...
+        for payment in model.payments:
+            total += payment.amount
+        total = total / 100
+        return Markup(total)
+
+    def _format_total_column_text_totalclaimed(view, context, model, name):
+        if model.status == model.STATE_DECLINED:
+            return Markup('-')
+        total = 0
```
Comment (djpnewton) [inline zapd/models.py]: unneeded variable

File: zapd/models.py [hunk 8]
```diff
@@ -315,11 +315,33 @@ def _format_total_column(view, context, model, name):
...
+            total += payment.amount
+        total = total / 100
+        return Markup(total)
+
+    def _format_total_column_text_totalclaimed(view, context, model, name):
+        if model.status == model.STATE_DECLINED:
+            return Markup('-')
+        total_claimed = 0
+        for payment in model.payments:
+            total += payment.amount
```
Comment (djpnewton) [inline zapd/models.py]: this will crash,

need to remove 'total' here

**Pull request merged**